CMakeLists.txt

```diff
@@ -243,6 +251,21 @@ if (HPGMP_ENABLE_HIP)
     target_link_libraries(hpgmp_device PRIVATE roc::rocprim)
     target_link_libraries(hpgmp PUBLIC hip::host roc::rocrand)
     target_link_libraries(hpgmp PUBLIC roc::rocblas roc::rocsparse)
+
+    if(HPGMP_ENABLE_PROFILING)
+      target_compile_definitions(hpgmp PUBLIC HPGMP_WITH_PROFILING)
+      target_include_directories(hpgmp PUBLIC 
+        ${ROCM_PATH}/include)
```

Comment:
We should not need this. FindHIP already adds these paths.
```suggestion
```

Reply:
Addressed in [a3d00b8](https://github.com/at-aaims/HPG-MxP/pull/3/commits/a3d00b811a5f9cee298f4b3b803a2543061aabdf)